src/vaults/BalanceSheet.sol

```diff
@@ -229,19 +245,26 @@ contract BalanceSheet is Auth, Recoverable, IBalanceSheet, IBalanceSheetGatewayH
         uint256 tokenId,
         address provider,
         uint128 amount,
-        D18 pricePoolPerAsset
+        D18 pricePoolPerAsset,
```

Comment:
QUESTION

No, blocker. But for follow-up. I would propose instead of adding all of that here, we should just trigger the message and the deposit call to the escrow directly FROM the `SyncRequyest` manager.

Reply:
But then the `BalanceSheet` is not the single source of truth for writing to the escrow anymore which I thought was one principe we wanted to follow. I suppose that changed given the latest work we did including this PR.

Would gladly let you do the follow up!